### PR TITLE
chore: generate confluent hub manifest

### PR DESCRIPTION
**Pull request by embano1**
Description
-----------

Add Confluent Maven plugin to generate `.zip` component file on each release which we submit to Confluent Hub so we're listed here https://www.confluent.io/hub/aws/kafka-eventbridge-sink

Test Steps
-----------

Checklist:
----------

- [x] I have tested my changes. No regression in existing tests.
- [ ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

Closes: #91

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by embano1** [comment]
We are live https://www.confluent.io/hub/aws/kafka-eventbridge-sink

@maschnetwork @baldawar can you please take a look at that page and let me know if you're 👍 ?
I'll discuss whether we can get a direct link/installation instructions like with [SQS](https://www.confluent.io/hub/confluentinc/kafka-connect-sqs) (from Confluent) and need to change the support level information on the left.

File: pom.xml [hunk 1]
```diff
@@ -275,6 +275,52 @@
...
+                            <documentationUrl>
+                                https://github.com/awslabs/eventbridge-kafka-connector#kafka-connector-for-amazon-eventbridge
+                            </documentationUrl>
+                            <logo>logos/eventbridge.jpg</logo>
+                            <ownerLogo>logos/aws.jpg</ownerLogo>
+                            <ownerName>Amazon Web Services</ownerName>
+                            <ownerType>organization</ownerType>
+                            <ownerUrl>https://aws.amazon.com</ownerUrl>
+                            <ownerUsername>aws</ownerUsername>
+                            <supportProviderName>Amazon Web Services</supportProviderName>
```
Comment (embano1) [inline pom.xml:307]: Remove

File: pom.xml [hunk 2]
```diff
@@ -275,6 +275,52 @@
...
+                            <supportSummary>The connector is released as a community-supported open-source project with
+                                best effort support from the repository maintainers.
+                            </supportSummary>
```
Comment (embano1) [inline pom.xml:310]: add `support.url`

**Comment by embano1** [comment]
@maschnetwork @baldawar page has been updated, can you please take a look?

Note that it currently says `Partner Supported` on the left side whereas the page itself states correctly that it's community supported. I asked Confluent and they said:

> yes that description is loaded into all connectors and cannot be modified per connector; but the support description on the connector indicates its community support as per your manifest

**Review by maschnetwork — APPROVED** [review]

**Pull request merged**